src/Symfony/Component/HttpFoundation/BinaryFileResponse.php

```diff
@@ -64,7 +64,7 @@ public function __construct($file, int $status = 200, array $headers = [], bool
      *
      * @throws FileException
      */
-    public function setFile($file, string $contentDisposition = null, bool $autoEtag = false, bool $autoLastModified = true)
+    public function setFile(\SplFileInfo|string $file, string $contentDisposition = null, bool $autoEtag = false, bool $autoLastModified = true)
```

Comment:
Couldn't comment on the line, but the `$file` parameter in the constructor can also become `\SplFileInfo | string $file`.